Program.cs

```diff
@@ -29,19 +29,18 @@ public static async Task Main(string[] args)
             /**
                 Create Database
             */
+
             try
             {
                 Console.WriteLine("Running Create Database API");
                 database = await databases.Create(
                     databaseId: ID.Unique(),
-                    name: "MoviesDB"
-                );
+                    name: "MoviesDB");
```

Comment:
Is this the right position for the closing bracket?

Reply:
This is the style recommended in the C# Coding Conventions